namespaces/api-description-namespace.md

```diff
@@ -177,13 +179,17 @@ Note: At the moment only the HTTP protocol is supported.
         Definition of any input message-body attribute for this transition.
 
     - `contentTypes` (array[String]) - A collection of content types that MAY be used for the transition.
-- `content` (array[HTTP Transaction]) - Array of transaction examples.
+- `content` (array)
+    - (Copy) - Textual information of the transition.
+    - (HTTP Transaction) - An instance of transaction example.
+
+        The `content` MAY include multiple `HTTP Transaction` elements.
```

Comment:
this is also the case for Copy, not sure if we need to say it as it should be clear from the MSON notation